crates/provider/src/fillers/mod.rs

```
@@ -329,7 +338,8 @@ where
     }
 
     fn call<'req>(&self, tx: &'req N::TransactionRequest) -> EthCall<'req, N, Bytes> {
-        self.inner.call(tx)
+        let prepare_res = self.prepare_call(tx.clone()).ok();
```

let's just ignore the error here for now

I think all of this needs some design changes but we can make his work without introducing the `filled_tx` fn and field

---

[e16d528](https://github.com/alloy-rs/alloy/pull/2092/commits/e16d5282b6cc7d2ae5b30152d967e99ee3dd6b46)